.github/workflows/_linux_ut.yml

```diff
@@ -79,9 +83,13 @@ jobs:
           else
             export _GLIBCXX_USE_CXX11_ABI=1
           fi
-          WERROR=1 python setup.py bdist_wheel
-          pip install --force-reinstall dist/*.whl
-          git clone https://github.com/pytorch/vision && cd vision && python setup.py install && cd ..
+          if [ ${{ inputs.test_type }} != "whl" ]; then
+            WERROR=1 python setup.py bdist_wheel
+            pip install --force-reinstall dist/*.whl
+            git clone https://github.com/pytorch/vision && cd vision && python setup.py install && cd ..
+          else
+            pip install torch torchvision torchaudio pytorch-triton-xpu --pre --index-url https://download.pytorch.org/whl/nightly/xpu
```

Comment:
let's keep standard nightky wheel installation cmd, remove the triton in the cmd, it's a dependency of torch.